### ISSUE TITLE
[Architecture] Multi-Site handling

### ISSUE THREAD

**Issue by ix5** [posativ/isso#891]
The current way Isso handles multi-site setups is quite confusing and causes many headaches internally. It has grown organically rather than being an intentionally designed feature.

We should decide whether we want to support multiple sites in a single instance (and admin) or if we want to continue the multi-instance `dispatch` "solution".

Issues:
- Migration of existing data and setups
- Handling multiple sites in the admin interface
- If using full URIs instead of only relative paths: Differentiating http/https variants

And much more to figure out.

**Comment by BBaoVanC** [comment]
Any way we can poll existing users for how many people actually use the multi-site feature? If it's low, then I feel that removing multi-site in favor of keeping Isso as a simple comment system would be best. And people can run multiple instances of Isso if they need multi-site.

I don't entirely understand how Isso handles multi-site internally, I'd have to go look at the code more to figure it out. But I get the feeling it's awkward.

**Comment by ix5** [comment]
> Any way we can poll existing users for how many people actually use the multi-site feature? If it's low, then I feel that removing multi-site in favor of keeping Isso as a simple comment system would be best.

Yes, surveying people of how they actually use Isso would be hugely beneficial. Maybe opening an issue or discussion and let people comment on it with their usecase. I feel like we could give some example points that we'd be interested in, but not constrain the answers too much or ask for too much redundant info. Of course, people active on the bugtracker are not representative of users as a whole.

**Comment by bestony** [comment]
I prefer multiple sites, not multiple instances, because I want configure Isso only once. (
